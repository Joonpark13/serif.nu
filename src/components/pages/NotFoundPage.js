import React from 'react';
import { Typography } from '@material-ui/core';
import { makeStyles } from '@material-ui/styles';
import { pageContainer, pageBody } from './common/styles';
import PageTitle from './common/PageTitle';

export const useStyles = makeStyles({
  pageContainer,
  pageBody,
});

export default function NotFoundPage() {
  const classes = useStyles();
  return (
    <div className={classes.pageContainer}>
      <PageTitle title="404 Page Not Found" />

      <div className={classes.pageBody}>
        <Typography variant="h5">
          The requested URL was not found on this server.
        </Typography>
      </div>
    </div>
  );
}
