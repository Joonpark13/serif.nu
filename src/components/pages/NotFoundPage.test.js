import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from './NotFoundPage';

describe('FAQPage', () => {
  it('should render correctly', () => {
    const wrapper = shallow(
      <NotFoundPage />,
    );

    expect(wrapper.get(0)).toMatchSnapshot();
  });
});
