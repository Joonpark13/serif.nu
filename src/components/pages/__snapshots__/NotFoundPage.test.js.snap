// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FAQPage should render correctly 1`] = `
<div
  className="makeStyles-pageContainer-1"
>
  <ForwardRef(WithStyles)
    title="404 Page Not Found"
  />
  <div
    className="makeStyles-pageBody-2"
  >
    <ForwardRef(WithStyles)
      variant="h5"
    >
      The requested URL was not found on this server.
    </ForwardRef(WithStyles)>
  </div>
</div>
`;
